src/functionalTest/resources/payloads/S-004.38_Existing_1.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Can you change the order of the payloads, so this will be the existing 2,  so it matches the behaviour of the original scenario - in matters of which one is in the db first. 
Also, when renaming use the format 'S-004.38_existingRecord_1.json' to keep it consistent with other file names.